### PR TITLE
Few more lines modified

### PR DESCRIPTION
**Pull request by ezzyventuresinc**
Please review modification advise/approve

### CODE REVIEW

File: file1 [hunk 1]
```diff
@@ -1,8 +1,29 @@
-We are enjoying git
-yet another line
+#We are enjoying git
```
Comment (enwokocha) [inline file1:1]: well done

File: file1 [hunk 2]
```diff
@@ -1,8 +1,29 @@
...
+#yet another line
 
 List of Fruits
 Apple
 Banana
 Orange
 Mango
+===============
```
Comment (enwokocha) [inline file1:9]: great!

**Review by enwokocha — APPROVED** [review]
Review Completed.

**Comment by ezzyventuresinc** [comment]
Thank you.

**Pull request merged**